components/squad/squad_row.lua

```diff
@@ -0,0 +1,82 @@
+local Class = require('Module:Class')
+local String = require('Module:String')
+local Player = require('Module:Player')
+
```

Comment:
Wiki-Custom version for Squad Row planned?

Reply:
Don't need to, this is it! Extend the class and implement as needed.